subworkflows/nf-core/bam_variant_demix_boot_freyja/main.nf

```diff
@@ -0,0 +1,75 @@
+include { FREYJA_VARIANTS   } from '../../../modules/nf-core/freyja/variants'
+include { FREYJA_UPDATE     } from '../../../modules/nf-core/freyja/update'
+include { FREYJA_DEMIX      } from '../../../modules/nf-core/freyja/demix'
+include { FREYJA_BOOT       } from '../../../modules/nf-core/freyja/boot'
+
+workflow BAM_VARIANT_DEMIX_BOOT_FREYJA {
+
+    take:
+    ch_bam          // channel: [ val(meta), [ /path/to/bam ] ]
+    fasta           // channel: [ val(meta), [path/to/reference.fasta] ]
+    repeats         // value repeats
+    db_name         // string db_name
+    barcodes        // /path/to/barcodes.csv
+    lineage_meta    // /path/to/lineage_metadata
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    //
+    // Variant calling
+    //
+    ch_freyja_variants = Channel.empty()
+    ch_freyja_depths    = Channel.empty()
+    FREYJA_VARIANTS(
+        ch_bam,
+        fasta)
+    ch_freyja_variants = FREYJA_VARIANTS.out.variants
+    ch_freyja_depths   = FREYJA_VARIANTS.out.depths
+
+    ch_versions=ch_versions.mix(FREYJA_VARIANTS.out.versions)
```

Comment:
```suggestion
    ch_versions=ch_versions.mix(FREYJA_VARIANTS.out.versions.first())
```

Can you add this to all versions mix channels? You only need one versions file per run for each module :)